client/js/screens/Faq/faq.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
Make sure your file name is FAQ not faq. Please make the changes.

Reply:
completed.